src/main/java/com/puppycrawl/tools/checkstyle/checks/coding/MissingSwitchDefaultCheck.java

```diff
@@ -88,6 +90,19 @@ public class MissingSwitchDefaultCheck extends AbstractCheck {
      */
     public static final String MSG_KEY = "missing.switch.default";
 
+    /**
+     * Represents the possible parent tokens of a switch statement.
+     */
+    private static final List<Integer> SWITCH_STATEMENT_PARENTS = List.of(
```

Comment:
Lets convert this type to Set. We just do search over this collection, so it will be slightly quicker.

Reply:
Done